pkg/terminal/terminalSesion.go

```diff
@@ -518,3 +504,25 @@ func getErrorMsg(err string) error {
 	}
 	return errors1.New(CommandExecutionFailed)
 }
+
+func (impl *TerminalSessionHandlerImpl) RunCmdInRemotePod(req *TerminalSessionRequest, cmds []string) (*bytes.Buffer, *bytes.Buffer, error) {
+	config, client, err := impl.getClientConfig(req)
+	if err != nil {
+		impl.logger.Errorw("error in fetching config", "err", err)
+		return nil, nil, err
+	}
+	impl.logger.Infow("reached getExecutor method call")
```

Comment:
do we need these, If Yes, then Please change Loglevel to ``Debug``

Reply:
done